### PR TITLE
CSPL-2179 QA test for PR https://github.com/splunk/splunk-operator/pull/1043

### PR DESCRIPTION
**Pull request by kashok-splunk**

### CODE REVIEW

**Review by pdhanoya-splunk — APPROVED** [review]

**Review by vivekr-splunk — APPROVED** [review]

File: test/monitoring_console/manager_monitoring_console_test.go [hunk 1]
```diff
@@ -937,4 +937,154 @@ var _ = Describe("Monitoring Console test", func() {
...
+			// Check standaloneOne Pod in MC Peer List
+			testcaseEnvInst.Log.Info("Check standalone  instance in MC Peer list")
+			testenv.VerifyPodsInMCConfigString(ctx, deployment, testcaseEnvInst, standalonePods, mcName, true, false)
+
+			// get revision number of the resource
+			resourceVersion := testenv.GetResourceVersion(ctx, deployment, testcaseEnvInst, mc)
+
+			// Check standaloneOne Pod in MC Peer List
+			testcaseEnvInst.Log.Info("Check standalone  instance in MC Peer list")
+			testenv.VerifyPodsInMCConfigString(ctx, deployment, testcaseEnvInst, standalonePods, mcName, true, false)
```
Comment (gaurav-splunk) [inline test/monitoring_console/manager_monitoring_console_test.go:1003]: Why is this needed again? We have already checked this in lines 994-996

Comment (kashok-splunk) [reply test/monitoring_console/manager_monitoring_console_test.go]: Fixed

**Review by gaurav-splunk — APPROVED** [review]
